docs/how-to/how-to-hotplug-memory-arm64.md

```diff
@@ -15,6 +15,19 @@ $ sudo .ci/aarch64/install_rom_aarch64.sh
 $ popd
 ```
 
+## Config KATA QEMU
+
+After executing the above script, two files will be generated under the directory /usr/share/kata-containers/ by default, namely kata-flash0.img and kata-flash1.img. Next we need to change the configuration file of kata qemu, which is in /opt/kata/share/defaults/kata-containers/configuration-qemu.toml by default, specify in the configuration file to use the UEFI ROM installed above. Please refer to the following configuration:
```

Comment:
Kata Containers has spelling checks for markdown, if the result shows some errors, you can add backticks to quote the words, especially for filenames or directories.

Reply:
Thanks for the tip, I'll adjust it based on follow-up results.